plugins/modules/dedicated_server_engagement_strategy.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import (absolute_import, division, print_function)
+
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: dedicated_server_engagement_strategy
+short_description: Sets the engagement strategy for a service
+description:
+    - This module sets the engagement strategy for a service
```

Comment:
Could you add some exemple for the engagement strategy, or maybe in the `options` below?
Or a link where ovh explains them?